src/main/kotlin/io/zerobase/smarttracing/resources/OrganizationsResource.kt

```diff
@@ -138,7 +142,8 @@ class OrganizationsResource(val dao: GraphDao, private val siteTypes: MultiMap<S
 
         val id = dao.createScannable(OrganizationId(orgId), SiteId(siteId), type, singleUse)
 
-        id?.let { amazon.scannableOrgEmail(dao.getEmailOrg(OrganizationId(orgId)), it.value) }
+        // TODO: Change null into bytearray for pdf
+        id?.let { email.send(dao.getEmailOrg(OrganizationId(orgId)), null, EmailType.CREATE_SCANNABLE) }
```

Comment:
this dao method should me renamed. at the least, it should be `getOrgEmail`, but there's not a great reason to shorten it so `getOrganizationEmail` would be better